src/vaults/SectorVault.sol

```diff
@@ -123,6 +132,8 @@ contract SectorVault is ERC4626, BatchedWithdraw {
 			strategy.deposit(address(this), address(asset), 0, param.minSharesOut);
 			totalStrategyHoldings += param.amountIn;
 		}
+
+		afterDeposit(0, 0);
```

Comment:
@0x0scion `depositIntoStrategies` logic will need to be updated.
I think we will want have an `availableDeposits` variable that tracks the amount of funds we are allowed to deposit into strategies. This is to ensure that we don't deposit funds sent by the cross-chain vault before we have accounted for them.

Additionally we need this variable to avoid spending funds earmarked for withdrawal.
(this is a note to self - no impact on PR)